### DISCUSSION TITLE
How to resolve problem of space

### DISCUSSION THREAD

**Discussion by pacdav** [supermerill/SuperSlicer#2011] [Q&A]
Hello
When I want to print some pieces which are specific like the octopus or other piece which can move. SuperSlicer block me and say that some pieces are to close. There is not enough space. How can I print that?
Thank
![SuperSlicer_2 3 57 a flavored version of Slic3r 05_12_2021 17_13_04](https://user-images.githubusercontent.com/49452344/144754332-ec5ccaed-fd27-4d84-b40d-f13b53f088a6.png)
s

**Comment by supermerill** [comment]
I think you want to print them one after the other, and not all at the same time?
I think it's because your extruder (print settings -> output) is too big, and may collide when printing another object.
If you are sure the height of the objects is lower than the nozzle height, then you may reduce the extruder radius to ~5mm for this print.

**Comment by neophyl** [comment]
Depending on your printer type its also a good idea to put the taller items at the back of the plate and order them so it prints the front ones first, moving front to back in general.
